scripts/kafka-integration-test.sh

```diff
@@ -3,9 +3,49 @@
 set -e
 
 export STORAGE=kafka
-while true; do
+
+# Function to start Kafka
+start_kafka() {
+    echo "Starting Kafka..."
+    
+    docker run --name kafka \
```

Comment:
this looks like a blocking run, don't you need to detach it? How did you test your changes?

Reply:
Here i am have added `-d` tag so the docker container will run in the background
```
docker run --name kafka -d\
--network jaeger \
    -p 9092:9092 \
    -e KAFKA_CFG_NODE_ID=0 \
    -e KAFKA_CFG_PROCESS_ROLES=controller,broker \
    -e KAFKA_CFG_CONTROLLER_QUORUM_VOTERS=0@kafka:9093 \
    -e KAFKA_CFG_LISTENERS=PLAINTEXT://:9092,CONTROLLER://:9093 \
    -e KAFKA_CFG_ADVERTISED_LISTENERS=PLAINTEXT://localhost:9092 \
    -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT \
    -e KAFKA_CFG_CONTROLLER_LISTENER_NAMES=CONTROLLER \
    -e KAFKA_CFG_INTER_BROKER_LISTENER_NAME=PLAINTEXT \
    bitnami/kafka:3.6
```

Reply:
`-d` is not in the PR

Reply:
Yes, i'll add it right away. Just waiting for confirmation.

Reply:
Getting an error while running `bash scripts/kafka-integration-test.yml`

![image](https://github.com/jaegertracing/jaeger/assets/107461226/98ec4c09-a71a-4e82-9cee-d2c508efadea)

Reply:
Getting an error. I think that error is related to a missing dependency.
![image](https://github.com/jaegertracing/jaeger/assets/107461226/075ec684-7c0b-4ae2-8c21-4d44a6882789)

Reply:
Acknowledged. I overlooked creating the network in the previous PR, resulting in test failures. The PR has been updated. Please review when you can. Thank you